plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetastoreAccessOperations.java

```diff
@@ -301,6 +302,12 @@ public void testSelectSystemTable()
                         .addCopies(GET_TABLE, 1)
                         .build());
 
+        // select from $metadata_log_entries
+        assertMetastoreInvocations("SELECT * FROM \"test_select_snapshots$metadata_log_entries\"",
+                ImmutableMultiset.<MetastoreMethod>builder()
+                        .addCopies(GET_TABLE, 1)
```

Comment:
nit:
```suggestion
                        .add(GET_TABLE)
```

Reply:
Just want to keep it consistent with the codes above and below.

Reply:
No need to use `addCopies` as @jkylling suggested. We can fix other places in follow-up. Same for `TestIcebergGlueCatalogAccessOperations`.